### PR TITLE
Traslate Spanish shop

### PR DESCRIPTION
**Pull request by Alexander-Luna**
## Documentation
- [ ] My pull request requires an update on the documentation repository.

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/lang/es/app.php [hunk 1]
```diff
@@ -3076,18 +3076,18 @@
                 'product-carousel-description'  => 'Muestra productos de forma elegante con un carrusel de productos dinámico y receptivo.',
                 'save-btn'                      => 'Guardar',
                 'select'                        => 'Seleccionar',
-                'slider'                        => 'Deslizador',
```
Comment (shivendra-webkul) [inline packages/Webkul/Admin/src/Resources/lang/es/app.php:3079]: No need to change this, it is already correct.

File: packages/Webkul/Admin/src/Resources/lang/es/app.php [hunk 2]
```diff
@@ -3076,18 +3076,18 @@
...
-                'slider-add-btn'                => 'Agregar Deslizador',
-                'slider-description'            => 'Personalización de temas relacionada con el deslizador.',
-                'slider-image'                  => 'Imagen del Deslizador',
-                'slider-required'               => 'El campo de deslizador es obligatorio.',
+                'slider'                        => 'Slider',
+                'slider-add-btn'                => 'Agregar Slider',
+                'slider-description'            => 'Personalización de temas relacionada con el slider.',
+                'slider-image'                  => 'Imagen del Slider',
+                'slider-required'               => 'El campo de slider es obligatorio.',
```
Comment (shivendra-webkul) [inline packages/Webkul/Admin/src/Resources/lang/es/app.php:3083]: No need to change this, it is already correct.

Comment (Alexander-Luna) [reply packages/Webkul/Admin/src/Resources/lang/es/app.php:3083]: I think the term used previously to refer to sliders was too ambiguous.

**Review by Alexander-Luna — COMMENTED** [review]
In spanish the sliders your name is sliders, i speak spanish native

**Comment by shivendra-webkul** [comment]
Hey @Alexander-Luna, it looks like your tests failed. Please run the pint command.

**Comment by devansh-webkul** [comment]
Please run pint command.

**Comment by devansh-webkul** [comment]
@Alexander-Luna,

Please resolve conflict and run the pint command.

**Pull request closed without merging**